developer_manual/getting_started/devenv.rst

```diff
@@ -87,6 +87,21 @@ or::
 
 Now access the installation at http://localhost/ (or the corresponding URL) in your web browser to set up your instance.
 
+Check out external shipped apps
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This optional step is especially necessary if you want to test upgrading, as the following apps are required to be present during an upgrade.
+
+Install the viewer app::
+
+  cd /var/www/apps
+  git clone https://github.com/nextcloud/viewer.git
```

Comment:
Maybe add a link to https://github.com/nextcloud/server/blob/master/core/shipped.json
This is the list people should try, and the `alwaysEnabled` is the must have array

Reply:
Updated, see last commit